src/components/form/input.js

```diff
@@ -191,7 +191,9 @@ const Input = ({
     const [is_password_visible, setPasswordVisible] = useState(false)
 
     useEffect(() => {
-        !is_password_visible ? (current_input.type = 'password') : (current_input.type = 'text')
+        !is_password_visible && password_icon
+            ? (current_input.type = 'password')
+            : (current_input.type = 'text')
```

Comment:
```
current_input.type = (!is_password_visible && password_icon) ? 'password' : 'text'
```

Reply:
Thanks for suggestion, looks more efficient, updated.